### PR TITLE
update efiling-frontend env setup

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

- Updates the .env.example with valid values
- Update to the README.md file

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/README.md [hunk 1]
```diff
@@ -4,31 +4,47 @@ Welcome to the eFiling frontend!
...
+```
 
 Installs all the required dependencies to get the application up and running.
 
-### `yarn start`
+```bash
+yarn start
+```
 
 Runs the app in the development mode.<br />
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/README.md:29]:
```suggestion
Runs the app in the development mode.

```

File: src/frontend/efiling-frontend/README.md [hunk 2]
```diff
@@ -4,31 +4,47 @@ Welcome to the eFiling frontend!
...
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
 The page will reload if you make edits.<br />
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/README.md:32]:
```suggestion
The page will reload if you make edits.

```

File: src/frontend/efiling-frontend/README.md [hunk 3]
```diff
@@ -4,31 +4,47 @@ Welcome to the eFiling frontend!
...
 You will also see any lint errors in the console.
 
-### `yarn run build`
+```bash
+yarn run build
+```
 
 Builds the app for production to the `build` folder.<br />
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/README.md:39]:
```suggestion
Builds the app for production to the `build` folder.

```

File: src/frontend/efiling-frontend/README.md [hunk 4]
```diff
@@ -4,31 +4,47 @@ Welcome to the eFiling frontend!
...
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.<br />
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/README.md:42]:
```suggestion
The build is minified and the filenames include the hashes.

```

**Review by alexjoybc — CHANGES_REQUESTED** [review]
nice thanks @KDNeufeld , just a couple formatting in the readme, html tags should be used as minimum as possible ;-)

File: src/frontend/efiling-frontend/README.md (hunk 1 above)
Comment (KDNeufeld) [reply src/frontend/efiling-frontend/README.md:29]: Sounds good. I only changed the yarn start lines, but these additional changes would also improve the readme.  Thanks!

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**